pkg/controller/issueapi/issue_batch.go

```diff
@@ -71,6 +72,13 @@ func (c *Controller) HandleBatchIssue() http.Handler {
 			return
 		}
 
+		if !membership.Can(rbac.CodeBulkIssue) {
```

Comment:
I think we can only do this check if it was called via the UI. There is no "membership" on the API.

I'm also pretty sure I did this - isn't there a specific handler for the UI server to server this?

Reply:
No the UI server remounts HandleIssue and HandleBulkIssue at another route (for ajax), but it's the same code.

What if we only check this if User is present (not authorized_app)

Reply:
I guess that's fine? I just really dislike when a code path behaves dramatically different based on some other data that's a few layers of indirection away.